scripts/xsslint/xsslint/linters.py

```diff
@@ -54,7 +54,7 @@ def _load_file(self, file_full_path):
         """
         with open(file_full_path, 'r') as input_file:
```

Comment:
Use 'io.open' here so that we always read the file as unicode and return the contents as a unicode string.  With just using open, it will provide the file as bytes in python2 and as unicode in python3 which will be inconsistent.